packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js

```diff
@@ -76,7 +76,9 @@ export default /* @ngInject */ ($stateProvider) => {
         VpsUpgradeService,
       ) =>
         isVpsNewRange
-          ? VpsUpgradeService.getAvailableUpgrades(serviceInfos.serviceId)
+          ? VpsUpgradeService.getAvailableUpgrades(
+              serviceInfos.serviceId,
+            ).catch(() => false)
```

Comment:
Same as https://github.com/ovh/manager/pull/8919/files#r1112784905

Reply:
Here i don't put [ ] because i have an if test. I putted null value i don't know if it is better.